Jenkinsfile

```diff
@@ -206,7 +206,7 @@ pipeline {
 
                 if (env.DEPLOY_SUCCESS == false) {
                     postGitHubStatus("jenkins/deploy", "failure", "Failed", env.RUN_DISPLAY_URL)
-                    postGitHubStatus("jenkins/functional-tests", "error", "Cancelled", env.JOB_URL)
```

Comment:
And there's no point to look at a log for a check that didn't run.

Reply:
Well, we still need the URL and we need to report an "error" status back to GitHub.

Reply:
`env.JOB_URL` is still a URL ;)

Reply:
Oh, I misunderstood from the diff — Yes, it is, but you can still see the failure logs directly for the prerequisite steps with `RUN_DISPLAY_URL`.

Reply:
Sorry, yeah, should have commented on the green line.

Fair point, they probably want those logs if the functional tests didn't run.